packages/cozy-harvest-lib/test/components/TriggerManager.spec.js

```diff
@@ -7,6 +7,19 @@ import cronHelpers from 'helpers/cron'
 import KonnectorJobWatcher from '../../src/models/konnector/KonnectorJobWatcher'
 import { KonnectorJobError } from 'helpers/konnectors'
 
+jest.mock('cozy-doctypes', () => {
+  return {
+    CozyFolder: {
+      registerClient: () => {},
+      ensureMagicFolder: () => ({ path: '/Administrative' }),
+      magicFolders: {
+        ADMINISTRATIVE: '/adminsitrative',
```

Comment:
Can we have something closer to reality? 

```
magicFolders. ADMINISTRATIVE : 'io.cozy.apps/administrative'
```

Should be easier to understand the difference between this magic folder and the path returned by `ensureMagicFolder`